app/frontend/app/templates/notifications.hbs

```diff
@@ -1,11 +1,35 @@
 <div class="container">
   <div class="row" style="margin-top: 50px">
-    <div class="col-md-6 col-md-offset-3">
+    <div class="col-md-10 col-md-offset-1">
       <ul class="all-notifications">
-        {{#each}}
-          {{partial "notification"}}
+        {{#each notification in model}}
+          <div {{bind-attr class=":notification-item notification.seen::new"}}>
+            <div class="notification-title">
+              <a {{bind-attr href=notification.link}}>
+                <div class="avatar">
+                  {{avatar notification.sourceUser 'thumb_small'}}
```

Comment:
I'm more thinking about cases where the source might be a show that just aired or your MAL import finishing